Server/controllers/authController.js

```
@@ -173,6 +173,68 @@ const loginUser = async (req, res, next) => {
+        error.service = SERVICE_NAME;
+        return next(error);
+      }
+    });
+
+    // Refresh token is valid and unexpired, generate new access token
+    const oldAuthToken = getTokenFromHeaders(req.headers);
+    const { jwtSecret } = req.settingsService.getSettings();
+
+    const payloadData = jwt.verify(oldAuthToken, jwtSecret, { ignoreExpiration: true });
+    // delete old token related data
+    delete payloadData.iat;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)